tests/clientTests/componentsTests/star.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Star from '../../../client/components/star';
+
+describe('Star components', () => {
+  const star = mount(<Star />);
+
+  test('Star component should contain the star image ip address', () => {
+    const image = 'https://s3-us-west-1.amazonaws.com/foodigouserphotos/Star.png';
```

Comment:
I'd consider loading this image from the directory just because it's so small. When we deploy later I think the amazon bucket will look in our project file?